src/apps/popup/pages/home/index.tsx

```diff
@@ -66,6 +73,7 @@ const BalanceContainer = styled(CenteredFlexColumn)`
 `;
 
 const ButtonsContainer = styled(SpaceAroundFlexColumn)`
+  flex-direction: row;
```

Comment:
better to create `SpaceAroundFlexRow` container for flex row instead of rewriting the container for the column into a row